bulbs/videos/templates/videos/series-page.html

```diff
@@ -0,0 +1,27 @@
+{% load betty core %}
+
+<header>
+  <h1 id="series-title"></h1>
+  <p>Watch The Latest Episode</p>
+</header>
+
+<figure class="video-container">
+  <div class="content">
+    <bulbs-video
+      twitter-handle="avclub"
+      src="{{ VIDEOHUB_BASE_URL }}/video/"
```

Comment:
That sounds correct, this will make a request to `//www.onionstudios.com/video/`